src/ADAL.PCL.Android/BrokerProxy.cs

```diff
@@ -371,7 +371,7 @@ private string GetRedirectUriForBroker()
             string signatureDigest = this.GetCurrentSignatureForPackage(packageName);
             if (!string.IsNullOrEmpty(signatureDigest))
             {
-                return string.Format(CultureInfo.CurrentCulture, "{0}://{1}/{2}", RedirectUriScheme,
+                return string.Format(CultureInfo.InvariantCulture, "{0}://{1}/{2}", RedirectUriScheme,
                     packageName.ToLower(), signatureDigest);
```

Comment:
Not sure what the rules about case-sensitivity are for Android package names. Leaving this.